libs/island-ui/contentful/src/lib/TellUsAStoryForm/TellUsAStoryForm.tsx

```diff
@@ -323,7 +323,6 @@ export const TellUsAStoryForm: React.FC<TellUsAStoryFormProps> = ({
                 <GridColumn span="12/12">
                   <Stack space={3}>
                     <Input
-                      name="subject"
```

Comment:
Quick note for review: 
These `name="whatever"` were throwing errors because there was a spread operator from a register method that used the same key that would generate a name later.
This therefore needs to be removed since it would've been overwritten, hence the typescript error.